lib/public_key/test/public_key_SUITE.erl

```diff
@@ -1355,6 +1373,6 @@ pss_params(sha256) ->
        maskGenAlgorithm = #'MaskGenAlgorithm'{algorithm = ?'id-mgf1',
                                               parameters = #'HashAlgorithm'{algorithm = ?'id-sha256'}
                                              },
-       saltLength = 32,
+       saltLength = 20,
```

Comment:
It is recommended that the MGF hash function be the same as the scheme hash algorithm/function and that the salt length is hLen, the length of the output of the hash function.